<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1634184915867' stop='1634184926524'>
    <name>angularjs homepage</name>
    <title>angularjs homepage</title>
    <test-cases>
        <test-case start='1634184915868' status='undefined'>
            <name>Addition of two numbers3</name>
            <title>Addition of two numbers3</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1634184915868' status='failed' stop='1634184926523'>
            <name>Addition of two numbers3</name>
            <title>Addition of two numbers3</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='08a80461-32d8-4b9f-887e-19e5cf620bf9-attachment.png' type='image/png' size='17544'/>
            </attachments>
            <failure>
                <message>Failed: invalid selector: Unable to locate an element with the xpath expression //div[@class='alert alert-success] because of the following error:
                SyntaxError: Failed to execute 'evaluate' on 'Document': The string '//div[@class='alert alert-success]' is not a valid XPath expression.
                  (Session info: chrome=94.0.4606.61)
                  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.17134 x86_64)</message>
                <stack-trace>InvalidSelectorError: invalid selector: Unable to locate an element with the xpath expression //div[@class='alert alert-success] because of the following error:
                SyntaxError: Failed to execute 'evaluate' on 'Document': The string '//div[@class='alert alert-success]' is not a valid XPath expression.
                  (Session info: chrome=94.0.4606.61)
                  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.17134 x86_64)
                    at Object.checkLegacyResponse (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)
                From: Task: WebDriver.findElements(By(xpath, //div[@class='alert alert-success]))
                    at Driver.schedule (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at Driver.findElements (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:1048:19)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\built\element.js:159:44
                    at ManagedPromise.invokeCallback_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as getText] (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as getText] (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (D:\synechron_11_10_Protractor\Training\protractor\development.js:17:61)
                    at C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                From: Task: Run it("Addition of two numbers3") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\synechron_11_10_Protractor\Training\protractor\development.js:3:3)
                    at addSpecsToSuite (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Aravind\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\synechron_11_10_Protractor\Training\protractor\development.js:1:1)
                    at Module._compile (internal/modules/cjs/loader.js:1063:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
                    at Module.load (internal/modules/cjs/loader.js:928:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:769:14)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>